features/gift_exchanges/notification_emails.feature

```diff
@@ -0,0 +1,60 @@
+Feature: Gift Exchange Notification Emails
+  Make sure that gift exchange notification emails are formatted properly
+
+  Scenario: Assignment notifications with linebreaks.
+    Given I have created the tagless gift exchange "Holiday Swap"
+      And I open signups for "Holiday Swap"
+      And I create an assignment notification message with linebreaks for "Holiday Swap"
+
+    When I am logged in as "participant1"
+      And I start signing up for "Holiday Swap"
+      And I press "Submit"
+    Then I should see "Sign-up was successfully created."
+
+    When I am logged in as "participant2"
+      And I start signing up for "Holiday Swap"
+      And I press "Submit"
+    Then I should see "Sign-up was successfully created."
+
+    When I close signups for "Holiday Swap"
+      And I have generated matches for "Holiday Swap"
+      And all emails have been delivered
```

Comment:
Will this work with the `And I have sent assignments for "Holiday Test"` step instead of these four lines?